src/Domain/ContentRenderer/RendererConfig.php

```diff
@@ -9,7 +9,8 @@ class RendererConfig {
 	public function __construct(
 		public string $fileExtension,
 		public string $language,
-		public bool $showLineNumbers
+		public bool $showLineNumbers,
+		public string $entryPoint
```

Comment:
`bool $showEditButton` is more appropriate

Reply:
Have used an EntryPoint variable instead of isBitBucket or showEditButton, as we might later also be able to use it for anything other we need to do that is specific to either BitBucket or Embed; at that point, we'll then be renaming this back to a generic name. If you feel there won't be any later use cases for knowing which EntryPoint was used, then I can go ahead and rename this.

Reply:
Things inside `Domain/` are not supposed to know about "entry points".

Reply:
Got it, changing it.

Reply:
Done, please review.